editors/code/src/highlighting.ts

```diff
@@ -84,17 +101,32 @@ export class Highlighter {
             if (!byTag.get(d.tag)) {
                 continue;
             }
-            byTag
-                .get(d.tag)!
-                .push(Server.client.protocol2CodeConverter.asRange(d.range));
+            
+            if (d.id) {
```

Comment:
To make this opt-in, add checking a settings flag here

Reply:
:taco: 

EDIT: that should be :+1:, github autocomplete is ... too smart?